samples/MvcSample.Web/LanguageViewLocationExpander.cs

```diff
@@ -1,69 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
```

Comment:
There's a copy of this expander in one of the functional tests too. Might want to remove that.